dotenv-linter/install.ps1

```diff
@@ -26,8 +26,7 @@ IF (!(Test-Path -Path "$Env:USERPROFILE\Downloads\$Env:WEBI_PKG_FILE"))
     & move "$pkg_download.part" "$pkg_download"
 }
 
-IF (!(Test-Path -Path "$pkg_src_cmd"))
```

Comment:
Only the IF blocks directly surrounding New-Item commands should have been removed, not ALL IF blocks.